mat64/vector.go

```diff
@@ -97,6 +102,10 @@ func (v *Vector) T() Matrix {
 	return Transpose{v}
 }
 
+// Reset zeros the length of the vector so that it can be reused as the
+// receiver of a dimensionally restricted operation.
+//
+// See the Reseter interface for more information.
 func (v *Vector) Reset() {
 	v.mat.Data = v.mat.Data[:0]
```

Comment:
Sorry, I meant the internal comment. Altered here to reflect the difference in the two types:

```
	// No change of Inc or n to 0 may be
	// made unless both are set to 0.
```